package/boot/arm-trusted-firmware-mvebu/Makefile

```diff
@@ -21,13 +21,21 @@ PKG_LICENSE_FILES:=docs/license.rst
 
 PKG_MAINTAINER:=Vladimir Vid <vladimir.vid@sartura.hr>
 
+PKG_TARGETS:=bin
+PKG_FLAGS:=nonshared
+
 include $(INCLUDE_DIR)/package.mk
 
 define Package/arm-trusted-firmware-mvebu
   SECTION:=boot
   CATEGORY:=Boot Loaders
   DEPENDS:=@TARGET_mvebu_cortexa53
   VARIANT:=$(subst arm-trusted-firmware-mvebu-,,$(1))
+  DEFAULT:=y if BUILDBOT
```

Comment:
Small nit. If any of the boards supported by OpenWrt has available bootloader, it's always built by default, so I don't think distinguishing between buildbot and own build is necessary.

Reply:
That's for u-boot and what BUILD_DEVICES does (which the patch `uboot-mvebu: don't default to enable a3700 builds` disables). atf never defaulted to y afaict, else we would already see published binaries.